fb2k/in_vgmstream.cpp

```diff
@@ -250,6 +252,11 @@ void input_vgmstream::decode_seek(double p_seconds,abort_callback & p_abort) {
 	int max_buffer_samples = sizeof(sample_buffer)/sizeof(sample_buffer[0])/vgmstream->channels;
 	bool loop_okay = loop_forever && vgmstream->loop_flag && !ignore_loop && !force_ignore_loop;
 
+	/* start on first channel to avoid problems in some decoders */
```

Comment:
What is the meaning of this? This sample count is in whole frames, not N samples per channel.